testutils/parsers.go

```diff
@@ -0,0 +1,31 @@
+package testutils
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/stretchr/testify/require"
+)
+
+// GetPubkeys provides n random public keys for testing.
```

Comment:
remove random from comment, they're not random in the sense of "generated on the spot", although randomly picked, they're still the same every time